houston/houston/servicer/servicer.py

```diff
@@ -0,0 +1,84 @@
+import re
+import grpc
+import logging
+
+from model import get_incident, get_incidents
+from protos.houston_pb2_grpc import APIServiceServicer
+from protos.houston_pb2 import (
+    GetIncidentResponse,
+    GetIncidentsResponse,
+    Incident,
+)
+log =logging.getLogger(__name__)
+
+class Servicer(APIServiceServicer):
+    def GetIncident(self, request, context):
+        log.info("calling get_incident()")
+        context.set_code(grpc.StatusCode.OK)
+        specific_incident = Incident()
+        incident=get_incident(request.slack_channel_id)
+
+        if incident == None:
+            specific_incident = None
+        else: 
+            for x in incident:
+                specific_incident = makeIncidentObject(x)
+ 
+        
+        return GetIncidentResponse(
+            incident=specific_incident
+        )    
+
+    def GetIncidents(self, request, context):
+        log.info("calling get_incidents()")
```

Comment:
```suggestion
        log.info(f"calling get_incidents({request.limit}, {request.active})")
```

I think passing the args here would help make the log have a bit more context.